optimum/habana/accelerate/state.py

```diff
@@ -48,7 +48,10 @@ def __init__(self, cpu: bool = False, **kwargs):
             from habana_frameworks.torch.distributed.hccl import initialize_distributed_hpu
 
             if int(os.environ.get("LOCAL_RANK", -1)) != -1 and not cpu:
-                world_size, rank, local_rank = initialize_distributed_hpu()
+                world_size = os.environ.get("WORLD_SIZE", None)
+                local_rank = os.environ.get("LOCAL_RANK", None)
+                rank = os.environ.get("RANK", None)
```

Comment:
@yuanwu2017 can you point out where these are set in the mpi and deepspeed case? I only remember deepspeed has these set, can you conform?
what test have you done for this?

Reply:
1. https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling#using-deepspeed
![image](https://github.com/huggingface/optimum-habana/assets/34643241/bf93ed06-13a3-439a-9b39-c5b3517ac5a1)

2. https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling#single-card-training-gpt2
![image](https://github.com/huggingface/optimum-habana/assets/34643241/11364aae-79fb-40c7-95b0-ba6a42bb1042)
3.https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling#multi-card-training-with-deepspeed-gpt-j
![image](https://github.com/huggingface/optimum-habana/assets/34643241/ed602ef9-7fb6-44ed-b9e9-bee7a283c62e)

4.https://github.com/huggingface/optimum-habana/tree/main/examples/language-modeling#multi-card-training-gpt2
![image](https://github.com/huggingface/optimum-habana/assets/34643241/8a3a7957-e3cd-483f-ac1c-843583d90bbc)